shipyard/manifest.json

```diff
@@ -0,0 +1,58 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "3ea4d492-9d1c-4dda-bb8a-e571eca3ce7d",
+  "app_id": "shipyard",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Send Shipyard logs to Datadog",
+    "title": "Shipyard",
+    "media": [
+      {
+        "media_type": "image",
+        "caption": "Shipyard logo",
+        "image_url": "images/shipyard.svg"
+      }
+    ],
+    "classifier_tags": [
+      "Category::Containers",
+      "Category::Developer Tools",
+      "Category::Testing",
+      "Submitted Data Type::Logs",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS",
+      "Offering::Integration"
```

Comment:
Please also include the data types that you are submitting into Datadog, or querying from Datadog:

`Submitted Data Type::Logs`
`Queried Data Type::Events`

Reply:
@anawishnoff thanks for pointing this out, let me check with my team

Reply:
@nbeck415 Hey Natalie, what issues are you facing? Happy to get on a call but I may be able to answer them quickly async.

This set of classifier tags is simply to provide metadata about your integration, and will be displayed on the integration tile. You can read more in our [docs here](https://docs.datadoghq.com/developers/integrations/check_references/#classifier-tags), but we ask all partners to include the types of data that they'll be pulling out of Datadog (queried data types), and the types of data that they'll be sending into Datadog as a part of these classifier tags.

Reply:
@anawishnoff thanks for the reply -- I am looking at the helm yaml for the integration and trying to figure out which data types we are querying and sending. is there a specific section in that yaml that I'll find the info?

Reply:
Not sure if that's present in your code files, but rather something that someone on the product side on your team may know about? I'd talk to whoever conceptualized the integration :)

Reply:
@anawishnoff I asked and it seems we are just sending shipyard logs, not querying any datadog data. I will update this and then we're good to merge?

Reply:
Yup, looks good from my end now, thanks! Once our engineering team reviews, we'll be able to merge this.